operations/security_and_authentication/Add_LDAP_User_Federation.md

```diff
@@ -48,9 +50,64 @@ LDAP user federation is not currently configured in Keycloak. For example, if it
+
+      ```bash
+      ncn-m001# yq write -i ./customizations.yaml 'spec.kubernetes.sealed_secrets.keycloak_users_localize.generate.data.(args.name==ldap_connection_url).args.value' "ldap://$LDAP"
+      ncn-m001# yq read ./customizations.yaml spec.kubernetes.sealed_secrets.keycloak_users_localize
+      apiVersion: bitnami.com/v1alpha1
+      kind: SealedSecret
+      metadata:
+        annotations:
+          sealedsecrets.bitnami.com/cluster-wide: "true"
+        creationTimestamp: null
+        name: keycloak-users-localize
+      spec:
+        encryptedData:
+          ldap_connection_url: AgAmdO19GaLs3Yr9apnJ/JDuQS+6yMC+LlZrPO8g+9UvF2+0X1TifH/bPb0Dw4VMN/2MURx/vvwJE2DTz9yajuW3YJEdwD4o6z/OZ/qLDxu2u+HZSwRnWLWK6ROTBGMP7r0zOdQIDoeeAZw03+d4/UmiBlTlJhl+DJzcS8VbfJV+neNcZ0p5zcM7skqI5NL4teNItHoeuITC2QQ+TRQc/XOrkj3JxvrzFEtEstJz8fXOUXBOwakhRRzUZl9aAYcT6raK3mPQDg14AkM4JVCeku+h6O4OOoOIygC8FzfrXy+LWE93UZjw/0ZM+c6bqOLd7odGto5EylLaV7HS9V8trUPfKExBbYqoRzm+IU9eG3k8Gr7ijT9hRhr1wiV73DFy5NnNB0uAjFClnPXbqntbnSScLxHgFqrnitKM19RzcuHFaZ0Hq0S0VPO8az8BL7jiCkhlxqT0WN6I5RerHPt0PocikKJ/S58a2dc8uGwMgyAPCcJwNXnh7qoA3pgL72kD292ReCYUz6XR51XQAaW1S5O/OaI3VKCCHZAw+qCgW3tCraL3mjzuTYSQQQsDFicxcgQZVz26S/9ATW+3g8btOZlsJ8aA9zpSGAf+uo2N/OKDCmU60fxFTvddcYDlWZ9ZXM1q9lnDTzz1T+QMCZ/f+c2El4CgNvrKJvLueduuagGNfrYoVgGoQ76e/WVi866d1a0=
+        template:
+          metadata:
+            annotations:
+              sealedsecrets.bitnami.com/cluster-wide: "true"
+            creationTimestamp: null
+            name: keycloak-users-localize
+      generate:
+        data: {}
+      ```
+
+      Remove the existing keycloak_users_localize and cray-keycloak Sealed Secrets from customizations.yaml, and then add the `generate:` sections back in, populated with the desired configuration.
+
+      For example:
 
-      Follow step 8 under the "Create Baseline System Customizations" header in the [Prepare Site Init](../../install/prepare_site_init.md) procedure.
+      ```bash
+            cray-keycloak:
+                generate:
+                  name: keycloak-certs
+                  data:
+                    - type: static_b64
+                      args:
+                        name: certs.jks
+                        value: /u3+7QAAAAIAAAAA5yXvSDt11bGXyBA9M2iy0/5i1Tg=
+        
+            keycloak_users_localize:
+                generate:
+                  name: keycloak-users-localize
+                  data:
+                    - type: static
+                      args:
+                        name: ldap_connection_url
+                        value: "ldaps://my_ldap.my_org.test"
+                    - type: static
+                      args:
+                        name: ldap_bind_dn
+                        value: "cn=my_admin"
+                    - type: static
+                      args:
+                        name: ldap_bind_credentials
+                        value: "my_ldap_admin_password"
+      ```
    
    2. Update the LDAP settings.
```

Comment:
The first part of this says to set the values for the keycloak_users_localize keys in the spec.kubernetes.sealed_secrets field, but that was just done in the previous step...